fmriprep/cli/parser.py

```diff
@@ -48,6 +48,9 @@ def _build_parser(**kwargs):
         'aroma_err_on_warn': (None, '24.0.0'),
         'bold2t1w_init': ('--bold2anat-init', '24.2.0'),
         'bold2t1w_dof': ('--bold2anat-dof', '24.2.0'),
+        'force_bbr': ('--force bbr', '26.0.0'),
+        'force_no_bbr': ('--force no-bbr', '26.0.0'),
+        'force_syn': ('--force syn-sdc', '26.0.0'),
```

Comment:
in the case of multiple replacements, will they stack?

Reply:
These just provide warning messages; they don't actually do the replacement for users.

Reply:
ah, i gave myself too much credit 😆